### ISSUE TITLE
Not working on Mac

### ISSUE THREAD

**Issue by shinokada** [golopot/tex-to-unicode#13]
I am using Firefox addon on Mac.
When I press option + w, it types the sum symbol. How can I use it on Mac?

**Comment by golopot** [comment]
Thank you for posting this. I can confirm the key binding is not working in Firefox + Mac. I don't think there is a way to make it work without touching the codes. And the way to solve it seems to be changing the key binding from "Alt + w" to something else. But I am not sure about this solution because doing so will cause friction to users who is already familiar to this key binding. Thoughts?

**Comment by shinokada** [comment]
Can you add `linux` to `commands`?

[Link.](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands)

```
"commands": {
  "toggle-feature": {
    "suggested_key": {
      "default": "Alt+W",
      "mac": "MacCtrl+Shift+W"
    },
    "description": "Send a 'toggle-feature' event"
  }
}
```

On Macs, "Ctrl" is interpreted as "Command", so if you actually need "Ctrl", specify "MacCtrl".

**Comment by myucesan** [comment]
Still no update on this?

**Comment by golopot** [comment]
Closed by ce6f8dfa17c48326478d3448680076efc43794e2

Shortcut for Mac is now `control + w`. You can change it at chrome://extensions/shortcuts .
